near-sdk/src/utils/mod.rs

```diff
@@ -0,0 +1,56 @@
+use crate::Balance;
+
+#[macro_export]
+macro_rules! log {
+    ($arg:tt) => {
+        crate::env::log($arg.as_bytes())
+    };
+    ($($arg:tt)*) => {
+        crate::env::log(format!($($arg)*).as_bytes())
+    };
+}
+
+/// Represents 1 yocto-NEAR token
+pub const YOCTO_NEAR: Balance = 1;
+
+/// Represents 1 micro-NEAR token or 1_000_000_000_000_000 yocto-NEAR tokens (10**15).
+pub const NANO_NEAR: Balance = 1_000_000_000_000_000;
+
+/// Represents 1 micro-NEAR token or 1_000_000_000_000_000_000 yocto-NEAR tokens (10**18).
+pub const MICRO_NEAR: Balance = 1_000_000_000_000_000_000;
+
+/// Represents 1 milli-NEAR token or 1_000_000_000_000_000_000_000 yocto-NEAR tokens (10**21).
+pub const MILLI_NEAR: Balance = 1_000_000_000_000_000_000_000;
+
+/// Represents 1 NEAR token or 1_000_000_000_000_000_000_000_000 yocto-NEAR tokens (10**24).
+pub const NEAR: Balance = 1_000_000_000_000_000_000_000_000;
```

Comment:
See my comment here: https://github.com/near/near-sdk-rs/issues/160#issuecomment-694146865

Reply:
Removing this code from this PR to later address with strict balances